### PR TITLE
New formulas for pyeclib dependencies

### PR DESCRIPTION
**Pull request by achester**
New formulas for the following libraries:
* gf-complete
* jerasure
* liberasurecode

### CODE REVIEW

File: Library/Formula/liberasurecode.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make"
+    system "make", "test"
+    system "make", "install"
+  end
+
+  test do
+    system "true"
```
Comment (dunn) [inline Library/Formula/liberasurecode.rb]: Can you add a test for this formula too? Thanks!

**Comment by achester** [comment]
liberasurecode doesn't install any binaries, only libaries. What's the standard in that case? A hello world that links against the libraries? Any examples you can point to?

    archer:local admin$ brew ls --verbose liberasurecode
    /usr/local/Cellar/liberasurecode/1.1.0/AUTHORS
    /usr/local/Cellar/liberasurecode/1.1.0/ChangeLog
    /usr/local/Cellar/liberasurecode/1.1.0/COPYING
    /usr/local/Cellar/liberasurecode/1.1.0/include/liberasurecode/alg_sig.h
    /usr/local/Cellar/liberasurecode/1.1.0/include/liberasurecode/config_liberasurecode.h
    /usr/local/Cellar/liberasurecode/1.1.0/include/liberasurecode/erasurecode.h
    /usr/local/Cellar/liberasurecode/1.1.0/include/liberasurecode/erasurecode_backend.h
    /usr/local/Cellar/liberasurecode/1.1.0/include/liberasurecode/erasurecode_helpers.h
    /usr/local/Cellar/liberasurecode/1.1.0/include/liberasurecode/erasurecode_helpers_ext.h
    /usr/local/Cellar/liberasurecode/1.1.0/include/liberasurecode/erasurecode_log.h
    /usr/local/Cellar/liberasurecode/1.1.0/include/liberasurecode/erasurecode_postprocessing.h
    /usr/local/Cellar/liberasurecode/1.1.0/include/liberasurecode/erasurecode_preprocessing.h
    /usr/local/Cellar/liberasurecode/1.1.0/include/liberasurecode/erasurecode_stdinc.h
    /usr/local/Cellar/liberasurecode/1.1.0/include/liberasurecode/erasurecode_version.h
    /usr/local/Cellar/liberasurecode/1.1.0/include/liberasurecode/liberasurecode_rs_vand.h
    /usr/local/Cellar/liberasurecode/1.1.0/include/liberasurecode/list.h
    /usr/local/Cellar/liberasurecode/1.1.0/include/liberasurecode/rs_galois.h
    /usr/local/Cellar/liberasurecode/1.1.0/include/liberasurecode/xor_code.h
    /usr/local/Cellar/liberasurecode/1.1.0/include/liberasurecode/xor_hd_code_defs.h
    /usr/local/Cellar/liberasurecode/1.1.0/INSTALL_RECEIPT.json
    /usr/local/Cellar/liberasurecode/1.1.0/lib/liberasurecode.1.dylib
    /usr/local/Cellar/liberasurecode/1.1.0/lib/liberasurecode.a
    /usr/local/Cellar/liberasurecode/1.1.0/lib/liberasurecode.dylib
    /usr/local/Cellar/liberasurecode/1.1.0/lib/liberasurecode_rs_vand.1.dylib
    /usr/local/Cellar/liberasurecode/1.1.0/lib/liberasurecode_rs_vand.a
    /usr/local/Cellar/liberasurecode/1.1.0/lib/liberasurecode_rs_vand.dylib
    /usr/local/Cellar/liberasurecode/1.1.0/lib/libnullcode.1.dylib
    /usr/local/Cellar/liberasurecode/1.1.0/lib/libnullcode.a
    /usr/local/Cellar/liberasurecode/1.1.0/lib/libnullcode.dylib
    /usr/local/Cellar/liberasurecode/1.1.0/lib/libXorcode.1.dylib
    /usr/local/Cellar/liberasurecode/1.1.0/lib/libXorcode.a
    /usr/local/Cellar/liberasurecode/1.1.0/lib/libXorcode.dylib
    /usr/local/Cellar/liberasurecode/1.1.0/NEWS
    /usr/local/Cellar/liberasurecode/1.1.0/README

**Comment by dunn** [comment]
Yeah, [`tinyxml.rb`](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/tinyxml.rb) is a pretty good example.

**Comment by apjanke** [comment]
Merged. Thank you for your contribution to Homebrew!

**Pull request closed without merging**